Invalid memory address when running `docker-machine config`

```
➜  demo_pq git:(master) docker-machine config      
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x99d1]

goroutine 16 [running]:
runtime.panic(0x692600, 0xac1c24)
	/usr/src/go/src/pkg/runtime/panic.c:279 +0xf5
main.(*Host).GetURL(0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/docker/machine/host.go:318 +0x91
main.getMachineConfig(0x208c52480, 0x5, 0x0, 0x0)
	/go/src/github.com/docker/machine/commands.go:502 +0x473
main.cmdConfig(0x208c52480)
	/go/src/github.com/docker/machine/commands.go:223 +0x32
github.com/codegangsta/cli.Command.Run(0x731c30, 0x6, 0x0, 0x0, 0x7d82d0, 0x27, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/docker/machine/Godeps/_workspace/src/github.com/codegangsta/cli/command.go:101 +0xc6f
github.com/codegangsta/cli.(*App).Run(0x208c52240, 0x208bbc000, 0x2, 0x2, 0x0, 0x0)
	/go/src/github.com/docker/machine/Godeps/_workspace/src/github.com/codegangsta/cli/app.go:130 +0x953
main.main()
	/go/src/github.com/docker/machine/main.go:128 +0x773

goroutine 17 [runnable]:
runtime.MHeap_Scavenger()
	/usr/src/go/src/pkg/runtime/mheap.c:507
runtime.goexit()
	/usr/src/go/src/pkg/runtime/proc.c:1445

goroutine 18 [runnable]:
bgsweep()
	/usr/src/go/src/pkg/runtime/mgc0.c:1976
runtime.goexit()
	/usr/src/go/src/pkg/runtime/proc.c:1445

goroutine 19 [runnable]:
runfinq()
	/usr/src/go/src/pkg/runtime/mgc0.c:2606
runtime.goexit()
	/usr/src/go/src/pkg/runtime/proc.c:1445

goroutine 20 [runnable]:
os/signal.loop()
	/usr/src/go/src/pkg/os/signal/signal_unix.go:19
created by os/signal.init·1
	/usr/src/go/src/pkg/os/signal/signal_unix.go:27 +0x32
```

This is the first command I ran after installing the 0.1.0-rc2 binary on macos mavericks (x86-64). The keys were all created successfully and are in `~/.docker`.

--- a/commands.go
+++ b/commands.go
@@ -540,14 +540,20 @@ func getMachineConfig(c *cli.Context) (*machineConfig, error) {
 		if err != nil {
 			log.Fatalf("error getting active host: %v", err)
 		}
+		if m == nil {
+			return nil, fmt.Errorf("There is no active host")
+		}
 		machine = m
 	} else {
 		m, err := store.Load(name)
+		fmt.Println(err)
 		if err != nil {
 			return nil, fmt.Errorf("Error loading machine config: %s", err)
 		}
 		machine = m
 	}
+
+	fmt.Println(machine)
 
 	caCert := filepath.Join(utils.GetMachineClientCertDir(), "ca.pem")
 	clientCert := filepath.Join(utils.GetMachineClientCertDir(), "cert.pem")

